app/controllers/users/invitations_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Users::InvitationsController < Devise::InvitationsController
   before_action :set_target_organisation, if: :super_admin?, only: %i(create new)
   before_action :delete_user_record, if: :user_should_be_cleared?, only: :create
-  after_action :confirm_new_user_membership, only: :update
+  after_action :confirm_new_user_membership, only: :update # rubocop:disable Rails/LexicallyScopedActionFilter
```

Comment:
Do you know why this comment is needed? If I understand correctly, `Rails/LexicallyScopedActionFilter` ["checks that methods specified in the filter's only or except options are explicitly defined in the controller." ](https://github.com/rubocop-hq/rubocop/pull/5177) and I can see `confirm_new_user_membership` is defined in this controller

Reply:
This inherits from a Devise controller, and the action is defined there I believe.